devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/kotlin/pom.xml-template.ftl

```diff
@@ -107,7 +107,7 @@
                 </executions>
                 <configuration>
                     <javaParameters>true</javaParameters>
-                    <jvmTarget>1.8</jvmTarget>
+                    <jvmTarget>${java_target}</jvmTarget>
```

Comment:
so if grok this right this change is a change in the "api" of the code generation that means the templates now *require* java_target to be set which will not be the case if you are using 1.3.0 plugin (@aloubyansky correct me if i'm wrong on this here).

So 1.3.0 plugin will pick up latest 1.3.x platform and when generation happens it will fail.

To remedy this the various .ftl files should check if java_target is set.
if my freemarker syntax memory is right its something like `${java_target!"11"}`

Reply:
> so if grok this right this change is a change in the "api" of the code generation that means the templates now _require_ java_target to be set which will not be the case if you are using 1.3.0 plugin (@aloubyansky correct me if i'm wrong on this here).

That is correct.

Reply:
I suppose we should choose 1.8 as the default to have a consistent behavior. I'll go do that and test.